### PR TITLE
Lpd 22572

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-22572

### CODE REVIEW

File: modules/apps/commerce/commerce-product-options-web/src/main/java/com/liferay/commerce/product/options/web/internal/display/context/CPSpecificationOptionDisplayContext.java [hunk 1]
```diff
@@ -69,6 +64,14 @@ public CPSpecificationOptionDisplayContext(
 		setDefaultOrderByCol("label");
 	}
 
+	public String getApiURL(CPSpecificationOption cpSpecificationOption) {
+		long cpSpecificationOptionId = (cpSpecificationOption == null) ? 0 :
+			cpSpecificationOption.getCPSpecificationOptionId();
+
+		return "/o/headless-commerce-admin-catalog/v1.0/specifications/" +
+			cpSpecificationOptionId + "/list-type-definitions";
+	}
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-options-web/src/main/java/com/liferay/commerce/product/options/web/internal/display/context/CPSpecificationOptionDisplayContext.java:73]: If unused, remove it

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@andrea-ale-sbarra Overall looks good to me, just 1 comment. Please check it, thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/test/playwright/tests/commerce/commerce-product-options-web/commerceProductSpecifications.spec.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+	await commerceProductSpecificationsPage.addNewProductSpecifications.fill(
+		'Specification-1'
+	);
+
+	await commerceProductSpecificationsPage.addDescriptionSpecifications.fill(
+		'Specification-1 Description'
+	);
+
+	await expect(commerceProductSpecificationsPage.keyContent).toHaveValue(
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/commerce-product-options-web/commerceProductSpecifications.spec.ts:39]: lower case

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**